lib/nexaas/async/collector/persist.rb

```diff
@@ -14,7 +14,7 @@ def save(opts={})
           def content_in_json(opts)
             {
               'scope_id' => opts[:scope_id],
-              'content' => opts[:content],
+              'content' => Base64.encode64(opts[:content].to_s),
```

Comment:
what if `opts[:content]` is not text, but binary? like an image file for example, or a zip file. will the `.to_s` mess it up? we should test this case also.